steps/browserstack-upload/0.1.2/step.yml

```diff
@@ -0,0 +1,69 @@
+title: browserstack-upload
+summary: |
+  Upload APK/IPA files to BrowserStack's App Automate
+description: |
+  Upload your build files to Browserstack for remote device testing. Upload your IPA/APK file to BrowserStack App Automation and get back the app's url for your remote device tests.
+website: https://github.com/marcelkalveram/bitrise-step-browserstack-upload
+source_code_url: https://github.com/marcelkalveram/bitrise-step-browserstack-upload
+support_url: https://github.com/marcelkalveram/bitrise-step-browserstack-upload/issues
+published_at: 2018-10-11T17:32:47.851909971+02:00
+source:
+  git: https://github.com/marcelkalveram/bitrise-step-browserstack-upload.git
+  commit: e77a113ffeb57c35255bec81178ad6ac934723a2
+host_os_tags:
+- osx-10.10
+project_type_tags:
+- ios
+- android
+- xamarin
+- react-native
+- cordova
+- ionic
+type_tags:
+- utility
+- test
+toolkit:
+  bash:
+    entry_file: step.sh
+deps:
```

Comment:
Should this step run only on the OSX stacks?
What about the Ubuntu (Android & Docker on Ubuntu) ones? 

If you want to use this step on Ubuntu as well please add the `apt-get` dependency list just like you did with the `brew`.

Reply:
@BirmacherAkos I haven't tested the other stacks. I guess IPA files won't build there anyway.
For the APKs though, I can give it a try and amend the step accordingly. Give me a day or two.